specification/tags.json

```diff
@@ -67,7 +67,11 @@
     }
   },
   {
-    "name": "Integrations"
+    "name": "Integrations",
+    "deprecated": true
```

Comment:
This does not show anything in the UI, we need to put it on the `paths` (check how we do it for the `Regions`)
```suggestion
```